### DISCUSSION TITLE
Dell THin Client as NAS?

### DISCUSSION THREAD

**Discussion by templeowls21** [blakeblackshear/frigate#14024] [Ask A Question]
I have the Frigate add-on up and running on HA with 10 cameras, and need to store my recordings. I didn't realize that I need a NAS though with HA and the options are bit steep for me.

However, I currently have both a 2TB HDD, 2TB m.2 SSD, and unused Dell Thin Client laying around. Could I hypothetically use the Thin Client + either the SSD/HDD as a NAS? Or is that a dumb idea?

This is all new for me so apologies if this is a stupid question

**Comment by NickM-27** [comment]
yes, you could do that, should work fine.

**Reply by templeowls21** [reply]
Thanks! Is there a widely accepted NAS software to install to it that you'd suggest? I've seen people suggest OpenMediaVault or TrueNAS

**Reply by NickM-27** [reply]
you don't really need NAS software, all you really need is to run it with the storage and install NFS.

**Comment by arhills** [comment]
Surely just add the storage to whatever you are running HA and Frigate on?
